src/Components/WebAssembly/DevServer/src/blazor-devserver.runtimeconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "runtimeOptions": {
+    "tfm": "net6.0",
+    "framework": {
+      "name": "Microsoft.AspNetCore.App",
+      "version": "6.0.0-preview.3.21167.1"
```

Comment:
Do we need to worry about this version baked in?

Reply:
Could we do something like this? 
https://github.com/dotnet/aspnetcore/blob/main/src/Components/WebAssembly/BlazorManifest/src/BlazorManifest.csproj#L25-L28

Reply:
Updated this to use a template file.